packages/flutter_tools/test/integration.shard/flutter_build_preview_sdk.dart

```diff
@@ -91,4 +93,40 @@ void main() {
         .childFile('app-debug.apk').existsSync(), true);
     },
   );
+
+  test(
+    'build succeeds when both example app and plugin target compileSdkPreview',
+        () async {
+      final File appBuildGradleFile = exampleAppDir.childDirectory('android').childDirectory('app').childFile('build.gradle');
+      // write a build.gradle with compileSdkPreview as `Tiramisu` which is a string preview version
```

Comment:
You probably meant to say "UpsideDownCake" here?

Reply:
Yep, thanks for pointing it out! Changed in both places